virtcontainers/kata_agent.go

```diff
@@ -2070,3 +2075,14 @@ func (k *kataAgent) cleanup(id string) {
 		k.Logger().WithError(err).Errorf("failed to cleanup vm share path %s", path)
 	}
 }
+
+func (k *kataAgent) save() (s persistapi.AgentState) {
+	s.ProxyPid = k.state.ProxyPid
+	s.URL = k.state.URL
+	return
```

Comment:
This is of course correct, but would look more "conventional" as:

```go
return persistapi.AgentState{
    ProxyPid: k.state.ProxyPid,
    URL: k.state.URL,
}
```

Same comment for `load()` below.

Reply:
Nice, let me do this.